File: HealthcareBooking/HealthcareModels/Models/HealthcareStaff/Doctor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HealthcareModels.Models.HealthcareStaff
{
    public class Doctor : Staff
    {
        public string Specialties { get; set; }
        public string MedicalLincenseNumber { get; set; }
        public int YearsOfExperience { get; set; }
        public bool IsAcceptingNewPatients { get; set; }
        public string AssignedDepartment { get; set; }
    }
}

File: HealthcareBooking/HealthcareBookingFrontend/src/types/Doctor.ts
export interface Doctor {
  specialties: string;
  medicalLincenseNumber: string;
  yearsOfExperience: number;
  isAcceptingNewPatients: boolean;
  assignedDepartment: string;
  staffId: string;
  name: string;
  description: string;
  type: "Doctor"; // for validation
  supportedBookingTypeIds: string[];
  supportedBookingTypes: string[];
}
